src/assets/scripts/client/constants/gameOptionConstants.js

```diff
@@ -151,5 +152,47 @@ export const GAME_OPTION_VALUES = [
                 value: 'no'
             }
         ]
+    },
+    {
+        name: GAME_OPTION_NAMES.RANGE_RINGS,
+        defaultValue: 'default',
+        description: 'Range rings',
+        help: '@erikquinn your help text here',
```

Comment:
`Radius of range rings, in nautical miles`

Reply:
done